site/docs/transform/aggregate.md

```diff
@@ -111,3 +114,5 @@ The supported **aggregation operations** are:
 | max       | The maximum field value.|
 | argmin    | An input data object containing the minimum field value.|
 | argmax    | An input data object containing the maximum field value.|
+
+Note: It is important you [`parse`](data.html#format) your data types explicitly, especially if you are likely to have `null` values in your dataset and automatic type inference will fail.
```

Comment:
I think this note should be listed in the subsection for aggregate transform.  (Since this subsection is for supported aggregation operations, which are applicable for aggregate in either encoding or transform. )

Reply:
Agreed. Changed.